resources/duckling/rules/ru.numbers.clj

```diff
@@ -121,18 +121,11 @@
   ;;
   
   "ordinals (first..19th)"
-  #"(?i)(первый|второй|третий|четвертый|пятый|шестой|седьмой|восьмой|девятый|десятый|одинадцатый|двенадцатый|тринадцатый|четырнадцатый|пятнадцатый|шестнадцатый|семнадцатый|восемнадцатый|девятнадцатый|двадцатый)"
+  #"(?i)(перв|втор|трет|четверт|пят|шест|седьм|восьм|девят|десят|одинадцат|двенадцат|тринадцат|четырнадцат|пятнадцат|шестнадцат|семнадцат|восемнадцат|девятнадцат|двадцат)?(-)?(ый|ой|ий|ая|ое|ья|ье|ую|ые|ых)"
   {:dim :ordinal
-   :value (get {"первый" 1 "второй" 2 "третий" 3 "четвертый" 4 "пятый" 5
-              "шестой" 6 "седьмой" 7 "восьмой" 8 "девятый" 9 "десятый" 10 "одинадцатый" 11
-              "двенадцатый" 12 "тринадцатый" 13 "четырнадцатый" 14 "пятнадцатый" 15 "шестнадцатый" 16
-              "семнадцатый" 17 "восемнадцатый" 18 "девятнадцатый" 19 "двадцатый" 20}
+   :value (get {"перв" 1 "втор" 2 "трет" 3 "четверт" 4 "пят" 5
+              "шест" 6 "седьм" 7 "восьм" 8 "девят" 9 "десят" 10 "одинадцат" 11
+              "двенадцат" 12 "тринадцат" 13 "четырнадцат" 14 "пятнадцат" 15 "шестнадцат" 16 "семнадцат" 17 "восемнадцат" 18 "девятнадцат" 19 "двадцат" 20}
               (-> %1 :groups first .toLowerCase))}
 
-  "ordinal (digits)"
-  #"0*(\d+)(-)?(ый|ой|ий)"
-  {:dim :ordinal
-   :value (read-string (first (:groups %1)))}  ; read-string not the safest
```

Comment:
Have been fixing it at home. Will try to send a PR over the weekend with ordinal for digits and add more examples. I'm a native Russian speaker btw (the finnish surname can be confusing), if you are wondering where to proofread it.